data/Data Science Projet_data1_v2.R

```diff
@@ -0,0 +1,48 @@
+##read files
+temperature_nsw <- read.csv("C:/Users/howar/OneDrive/Documents/Data Science Project/temperature_nsw.csv", header=T);
```

Comment:
conscious that this code only works for you currently. I can adjust this to a dynamic directory rather than a hard-coded one